test/suites/migration.sh

```diff
@@ -183,6 +183,19 @@ migration() {
   lxc query /1.0/storage-pools/"${pool}"/volumes/container/cccp | jq .created_at | grep -Fv '0001-01-01T00:00:00Z'
   lxc query /1.0/storage-pools/"${pool}"/volumes/container/cccp/snapshots/snap0 | jq .created_at | grep -Fv '0001-01-01T00:00:00Z'
 
+  # When using zfs (particularly with xfs), wait for volume to be unmounted.
```

Comment:
I'd like to know more about this. As it feels like we're testing for bug here. Does this same issue exist for LVM (with xfs)?

Reply:
Which was the operation that would have triggered the unmount we are waiting for here? Shouldn't that not return until the unmount has finished?

Reply:
> Does this same issue exist for LVM (with xfs)?

I don't know.

> Which was the operation that would have triggered the unmount we are waiting for here? Shouldn't that not return until the unmount has finished?

I investigated this, and found that the following code snippet fails:
```
  echo test | lxc_remote file push - l1:c1/tmp/foo
  lxc_remote copy l1:c1 l2:c1
```

This seems to be caused by forkfile still running while attempting to copy. @stgraber proposed [this fix](https://gist.github.com/stgraber/4133fe08df647b9fac4ce3d96b8a7b22) which I'm currently testing.